app/cdap/components/shared/RichToolTip/RichToolTip.tsx

```diff
@@ -0,0 +1,175 @@
+/*
+ * Copyright © 2022 Cask Data, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+import {
+  Box,
+  ClickAwayListener,
+  Fade,
+  makeStyles,
+  Paper,
+  Popper,
+  PopperPlacementType,
+} from '@material-ui/core';
+import React, { useState, ReactElement, cloneElement } from 'react';
+
+interface IProps {
+  content: ReactElement;
+  children: ReactElement;
+  open: boolean;
+  onClose?: () => void;
+  arrow?: boolean;
+  placement?: PopperPlacementType;
+}
+
+const useStyles = makeStyles((theme) => {
```

Comment:
I thought we should avoid using `makeStyles`?

Reply:
If you look in the comments on this I basically copied that file from somewhere online - this won't be necessary with the new version of material ui since it has a much better api for its popover component. I don't want to spend time worrying about these styles since it can be replaced in the future and its a copied component